src/rust/aggregator/src/data/user_history.rs

```diff
@@ -56,7 +59,10 @@ impl Data for UserHistory {
             .begin()
             .await
             .map_err(|e| DataAggregationError::ProcessingError(anyhow!(e)))?;
-        let mut fill_events = sqlx::query!(
+        transaction.execute("SET TRANSACTION ISOLATION LEVEL REPEATABLE READ;")
+            .await
+            .map_err(|e| DataAggregationError::ProcessingError(anyhow!(e)))?;
```

Comment:
Actually, can we change this to `SERIALIZABLE`?

`REPEATABLE READ` has possible serialization anomolies in postgres: https://www.postgresql.org/docs/current/transaction-iso.html

And the aggregator is assumed to be effectively serial for this data stream